documentation/cloud-manager-guide/admin/user-management/delete-user.md

```diff
@@ -0,0 +1,48 @@
+---
+keywords:
+title: Delete a User
+description: Delete a user
+---
+# Delete a User
+
+This guide outlines the required steps to delete a user in Environment Operations Center.
+
+> **Warning:** Deleting a user is a permanent operation and cannot be undone.
+
+If you would prefer to deactivate a user, see the [edit a user](edit-user.md) guide for details on deactivating a user's account.
+
+## Getting started
+
+To delete a user, locate the user from the list of users displayed on the *Users* tab in the *Admin* section.
+
+![image description](images/delete-select-user.png)
+
+Select the **Options** (**menu**) associated with the user. From the options dropdown, select **Delete** to open the *Delete User* confirmation dialog.
+
+![image description](images/delete-options.png)
+
+## Delete user
+
+The *Delete User* dialog displays the email address associated with the user you are deleting and a warning message that the action cannot be undone.
+
+To proceed with deleting the user, enter the email address associated with their account in the space provided and select **Delete**.
+
+![image description](images/delete-dialog.png)
+
+To keep the user and exit out of the *Delete User* dialog, select **Keep The User**. You will return to the admin *Users* tab.
```

Comment:
Let's move this up to line 29 and then just have one image. It's a pretty easy concept, so I don't think we need two images here and this single line on its own.